runtime/util_core/j9argscan.c

```diff
@@ -385,6 +385,104 @@ scan_hex_caseflag_u64(char **scan_start, BOOLEAN uppercaseAllowed, uint64_t* res
 	return bits;
 }
 
+/**
+ * Scan the next unsigned number off of the argument string, and parses for memory sizes.
+ * Specify the size in GiBs, MiBs, or KiBs (with G,g,M,m,K,k suffixes) or in bytes (no suffix).
```

Comment:
please add `T` to the comment

Reply:
Thanks. See https://github.com/AlenBadel/openj9/commit/f45cc3f882c9f2f62b58c05119d54cfc759d8fc7